design-proposals/primary_network_binding_for_seamless_migration.md

```diff
@@ -0,0 +1,520 @@
+| Deployment                  | Binding Plugin                                                                                                  | Core                                 | Binding Plugin                                   |
+| IP & Route delivery         | Self (IPv4 & IPV6)                                                                                              | Self for IPv4, Net Provider for IPv6 | Net Provider                                     |
+| Pod Probes Support          | Yes                                                                                                             | No                                   | No                                               |
+| Seamless Live-Migration     | Requires Development                                                                                            | Yes                                  | Yes                                              |
+| Supported Traffic           | TCP/UDP/ICMP (All IP in case seamless migration is resolved)                                                    | IP                                   | IP                                               |
+| Migration Support Indicator | Yes                                                                                                             | Requires Development                 | Yes                                              |
+| Mesh / Istio support        | Yes                                                                                                             | No                                   | No                                               |
+| IP Status (reporting)       | Yes                                                                                                             | Yes                                  | Requires Development                             |
+| Observability Integration   | Yes                                                                                                             | No                                   | No                                               |
+| Resource Overhead           | 500MB per VM, 250MB per VM already available on U/S, Negligible overhead when resolving the seamless migration. | Negligible                           | Negligible                                       |
+| Performance                 | N/A                                                                                                             | N/A                                  | N/A , Most likely better than the bridge option. |
+| Kubevirt Support            | Low                                                                                                             | High                                 | Medium                                           |
```

Comment:
This entry is misleading: if bridge is deployed as a plugin, your maintenance costs will be quite lower.

You also don't mention the migration indicator (which I am unsure is required ...) is also not called out as a difference.

I think you should mention that in the callout below.

Reply:
@EdDev Can you please elaborate on the meaning of Kubevirt support? how do you measure it?

Reply:
@EdDev By `high` do you mean the boilerplate of SELinux re-labeling and the cgroup adjustment?